app/poros/video_results.rb

```diff
@@ -0,0 +1,7 @@
+class VideoResults
+  def create_videos(category)
+    video_service = VideoService.new
+    videos = video_service.get_videos(category)
+    videos[:data].map { |yoga_vid| Video.new(yoga_vid) }
```

Comment:
The naming of yoga_vid leads me to assume all the videos are yoga related? Is that the case?

Reply:
I think that is the category we started with so we should rename it to just videos for clarity.